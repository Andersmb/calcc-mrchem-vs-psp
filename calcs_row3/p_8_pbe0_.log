Starting job 5688132 on c1-[21,26],c3-[13,27],c5-22,c10-5 at Fri May 20 05:54:11 CEST 2022

Copying p_8_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying p_8_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688132       p_8_pbe0_        180                                             00:45:26      0:0 
5688132.bat+      batch         24        1   00:00:04          0   00:00:04   00:45:26      0:0 
5688132.ext+     extern        180        6   00:00:00          5   00:00:00   00:45:26      0:0 
5688132.0      mrchem.x        180       30   00:45:00         29   02:03:26   00:45:04      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688132                                                                          
5688132.bat+     19336K          0     19336K        0              0          0 
5688132.ext+          0          5          0        0              5          0 
5688132.0     16276164K         29 6757515400        0             25          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688132                                                                                               
5688132.bat+       16.90M               0         16.90M        0.03M                0          0.03M 
5688132.ext+        0.00M               5          0.00M            0                5              0 
5688132.0           7.32M              27          6.81M      145.41M                8         78.96M 

Job 5688132 completed at Fri May 20 06:39:38 CEST 2022
