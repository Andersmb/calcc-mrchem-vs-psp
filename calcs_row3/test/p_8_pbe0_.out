                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.876518        0.000000        0.000000
    1       F            :       -0.476337        2.565870        0.000000
    2       F            :       -0.476337       -1.282935        2.222129
    3       F            :       -0.476337       -1.282935       -2.222129
---------------------------------------------------------------------------
 Center of mass          :       -0.000000       -0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              21
 Beta electrons          :                                              21
 Total electrons         :                                              42
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              1.000000023718e+00
    1    1    a          :                              1.000000000965e+00
    2    1    a          :                              9.999987270665e-01
    3    1    a          :                              1.000000008391e+00
    4    1    a          :                              1.000000009430e+00
    5    1    a          :                              9.999999897506e-01
    6    1    a          :                              9.999999980425e-01
    7    1    a          :                              1.000000005957e+00
    8    1    a          :                              1.000000113728e+00
    9    1    a          :                              9.999998037400e-01
   10    1    a          :                              9.999999176740e-01
   11    1    a          :                              9.999996484436e-01
   12    1    a          :                              1.000000455207e+00
   13    1    a          :                              9.999994260036e-01
   14    1    a          :                              9.999992093480e-01
   15    1    a          :                              9.999999484282e-01
   16    1    a          :                              9.999997975171e-01
   17    1    a          :                              1.000000873582e+00
   18    1    a          :                              9.999998713260e-01
   19    1    a          :                              1.000001152440e+00
   20    1    a          :                              1.000000427792e+00
   21    1    b          :                              9.999999967577e-01
   22    1    b          :                              9.999999930575e-01
   23    1    b          :                              9.999999172160e-01
   24    1    b          :                              9.999998034519e-01
   25    1    b          :                              1.000000021630e+00
   26    1    b          :                              9.999999917844e-01
   27    1    b          :                              9.999999980425e-01
   28    1    b          :                              1.000000005903e+00
   29    1    b          :                              1.000000101581e+00
   30    1    b          :                              9.999998916046e-01
   31    1    b          :                              9.999999107841e-01
   32    1    b          :                              9.999996224926e-01
   33    1    b          :                              1.000000477968e+00
   34    1    b          :                              9.999993595788e-01
   35    1    b          :                              1.000000853762e+00
   36    1    b          :                              1.000001193081e+00
   37    1    b          :                              9.999999729090e-01
   38    1    b          :                              1.000000854989e+00
   39    1    b          :                              9.999997905655e-01
   40    1    b          :                              1.000001216981e+00
   41    1    b          :                              1.000000423980e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                634.888582100548
 E-N energy              :            (au)              -1906.699426550423
 Coulomb energy          :            (au)                494.971799493927
 Exchange energy         :            (au)                -13.278989493014
 X-C energy              :            (au)                -41.218246746696
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                194.299812920315
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -831.336281195659
 Nuclear energy          :            (au)                194.299812920315
---------------------------------------------------------------------------
 Total energy            :            (au)             -6.370364682753e+02
                         :      (kcal/mol)             -3.997464159411e+05
                         :        (kJ/mol)             -1.672539005383e+06
                         :            (eV)             -1.733464462913e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.027967934849
    1    1    a          :            (au)                -24.538848842659
    2    1    a          :            (au)                -24.537325061067
    3    1    a          :            (au)                -24.532830947173
    4    1    a          :            (au)                 -6.861704042259
    5    1    a          :            (au)                 -5.002131316930
    6    1    a          :            (au)                 -5.001834961301
    7    1    a          :            (au)                 -4.993134076346
    8    1    a          :            (au)                 -1.303412023663
    9    1    a          :            (au)                 -1.224271258452
   10    1    a          :            (au)                 -1.223179009017
   11    1    a          :            (au)                 -0.656217433002
   12    1    a          :            (au)                 -0.524665737091
   13    1    a          :            (au)                 -0.523240671027
   14    1    a          :            (au)                 -0.512557222125
   15    1    a          :            (au)                 -0.426480975562
   16    1    a          :            (au)                 -0.426162326755
   17    1    a          :            (au)                 -0.400001897244
   18    1    a          :            (au)                 -0.398784904327
   19    1    a          :            (au)                 -0.381736157920
   20    1    a          :            (au)                 -0.305826464815
   21    1    b          :            (au)                -77.027974501912
   22    1    b          :            (au)                -24.538857350292
   23    1    b          :            (au)                -24.537324085172
   24    1    b          :            (au)                -24.532821282878
   25    1    b          :            (au)                 -6.861704156405
   26    1    b          :            (au)                 -5.002131044727
   27    1    b          :            (au)                 -5.001834707791
   28    1    b          :            (au)                 -4.993133529622
   29    1    b          :            (au)                 -1.303404382038
   30    1    b          :            (au)                 -1.224274123721
   31    1    b          :            (au)                 -1.223176011440
   32    1    b          :            (au)                 -0.656221133736
   33    1    b          :            (au)                 -0.524686595338
   34    1    b          :            (au)                 -0.523254996182
   35    1    b          :            (au)                 -0.512557519524
   36    1    b          :            (au)                 -0.426488407283
   37    1    b          :            (au)                 -0.426170915906
   38    1    b          :            (au)                 -0.399998304560
   39    1    b          :            (au)                 -0.398780849997
   40    1    b          :            (au)                 -0.381733583003
   41    1    b          :            (au)                 -0.305823831759
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -361.604664576869
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          6.480741e+00        -637.036468275343         -6.370365e+02
    1          4.088057e-01        -640.572026355443         -3.535558e+00
    2          1.637984e-01        -640.716037710466         -1.440114e-01
    3          9.376276e-02        -640.742550925164         -2.651321e-02
    4          4.978159e-02        -640.750774011924         -8.223087e-03
    5          5.351883e-02        -640.757086108221         -6.312096e-03
    6          1.604021e-02        -640.757788807772         -7.026996e-04
    7          1.526442e-02        -640.758233423362         -4.446156e-04
    8          4.830382e-03        -640.758284022533         -5.059917e-05
    9          4.075042e-03        -640.758308404684         -2.438215e-05
   10          1.591626e-03        -640.758313782873         -5.378189e-06
   11          1.608865e-03        -640.758317227283         -3.444410e-06
   12          7.249391e-04        -640.758317963733         -7.364495e-07
   13          3.976334e-04        -640.758318273334         -3.096012e-07
   14          2.886355e-04        -640.758318429172         -1.558379e-07
   15          1.697386e-04        -640.758318485223         -5.605125e-08
   16          7.717820e-05        -640.758318501453         -1.622993e-08
---------------------------------------------------------------------------
                      SCF converged in 16 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.876518        0.000000        0.000000
    1       F            :       -0.476337        2.565870        0.000000
    2       F            :       -0.476337       -1.282935        2.222129
    3       F            :       -0.476337       -1.282935       -2.222129
---------------------------------------------------------------------------
 Center of mass          :       -0.000000       -0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                639.656126324552
 E-N energy              :            (au)              -1915.840454170568
 Coulomb energy          :            (au)                495.523683597661
 Exchange energy         :            (au)                -13.238354968537
 X-C energy              :            (au)                -41.159132204877
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                194.299812920315
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -835.058131421768
 Nuclear energy          :            (au)                194.299812920315
---------------------------------------------------------------------------
 Total energy            :            (au)             -6.407583185015e+02
                         :      (kcal/mol)             -4.020819122002e+05
                         :        (kJ/mol)             -1.682310721737e+06
                         :            (eV)             -1.743592132873e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.349099409848
    1    1    a          :            (au)                -24.818998856364
    2    1    a          :            (au)                -24.818996373903
    3    1    a          :            (au)                -24.818990942232
    4    1    a          :            (au)                 -6.734286818116
    5    1    a          :            (au)                 -4.871483176467
    6    1    a          :            (au)                 -4.871483065851
    7    1    a          :            (au)                 -4.864875369843
    8    1    a          :            (au)                 -1.337009519087
    9    1    a          :            (au)                 -1.283257301389
   10    1    a          :            (au)                 -1.283254355139
   11    1    a          :            (au)                 -0.704746186872
   12    1    a          :            (au)                 -0.600721633966
   13    1    a          :            (au)                 -0.600720829843
   14    1    a          :            (au)                 -0.567020169833
   15    1    a          :            (au)                 -0.528933324094
   16    1    a          :            (au)                 -0.528932176066
   17    1    a          :            (au)                 -0.488803043376
   18    1    a          :            (au)                 -0.488801258335
   19    1    a          :            (au)                 -0.472327073961
   20    1    a          :            (au)                 -0.320530223523
   21    1    b          :            (au)                -77.349099410299
   22    1    b          :            (au)                -24.818998857119
   23    1    b          :            (au)                -24.818996372945
   24    1    b          :            (au)                -24.818990943882
   25    1    b          :            (au)                 -6.734286819512
   26    1    b          :            (au)                 -4.871483177394
   27    1    b          :            (au)                 -4.871483064879
   28    1    b          :            (au)                 -4.864875370422
   29    1    b          :            (au)                 -1.337009519327
   30    1    b          :            (au)                 -1.283257302064
   31    1    b          :            (au)                 -1.283254354777
   32    1    b          :            (au)                 -0.704746187025
   33    1    b          :            (au)                 -0.600721633995
   34    1    b          :            (au)                 -0.600720829517
   35    1    b          :            (au)                 -0.567020169947
   36    1    b          :            (au)                 -0.528933323392
   37    1    b          :            (au)                 -0.528932176685
   38    1    b          :            (au)                 -0.488803043913
   39    1    b          :            (au)                 -0.488801258354
   40    1    b          :            (au)                 -0.472327073997
   41    1    b          :            (au)                 -0.320530223481
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -364.706542221035
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.127513        0.000023       -0.000006
 Magnitude               :            (au)                        0.127513
                         :         (Debye)                        0.324105
---------------------------------------------------------------------------
 Nuclear vector          :        0.286667        0.000000        0.000000
 Magnitude               :            (au)                        0.286667
                         :         (Debye)                        0.728635
---------------------------------------------------------------------------
 Total vector            :        0.159154        0.000023       -0.000006
 Magnitude               :            (au)                        0.159154
                         :         (Debye)                        0.404530
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 45m  1s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
